### DISCUSSION TITLE
LCD and .syx files problems

### DISCUSSION THREAD

**Discussion by JoaquinSainz** [probonopd/MiniDexed#190] [Q&A]
Hi everybody, just starting to put together this great project.  I have a problem with the LCD since the contrast has very little brightness, do you know what is the reason?
I also have a problem that I can't find the .syx files that I have added to the sd card, I put them in a folder in the way that Folyd Steinberg does in his tutorial. 
I hope you can help me guys, have a nice day ✌️
![image](https://user-images.githubusercontent.com/43962625/165891785-20748c40-a052-466f-a9d7-6e431279d008.jpeg)

**Comment by craigyjp** [comment]
I cannot see any connections to your LCD backlight.

Do you have the LCD backlight connected to +5v and ground? Preferably with a 10k ohm pot to control the voltage to the backlight so you can adjust it.

**Reply by arsamus** [reply]
> I cannot see any connections to your LCD backlight.
> 
> Do you have the LCD backlight connected to +5v and ground? Preferably with a 10k ohm pot to control the voltage to the backlight so you can adjust it.

As I see on the picture you miss to connect A (+5v) and K(GND) pins. This feed the backlight LED light.

**Comment by probonopd** [comment]
Would be interesting to see the back side of your board.

Do you have the 2.2k contrast resistor from LCD pin 3 (=V0) to GND installed, or a 10k potentiometer? Looks like yes.

Do you have LCD power connected? Looks like not. __Caution:__ Make sure you have a **LED current limiting resistor** in the circuit there! (May need a multimeter to verify.)

See my photo in https://github.com/probonopd/MiniDexed/issues/163.

**Comment by diyelectromusic** [comment]
Without seeing a directory listing of your syx files it isn't easy to see what is wrong, but if you connect up a monitor to the HDMI temporarily you will get some diagnostics that might give you a clue.

A key fault I initially had was not having the proper filename structure of nnnnnn_BankName and then in another case I realised I'd called the directory sysex/voices rather than sysex/voice.  The wiki "files" area now talks about this is a bit more detail.

There are a few other ideas of things that might go wrong in my own notes here: https://diyelectromusic.wordpress.com/2022/04/28/bare-metal-raspberry-pi-minidexed-dx7/

Kevin.

**Reply by krunchr** [reply]
> There are a few other ideas of things that might go wrong in my own notes here: https://diyelectromusic.wordpress.com/2022/04/28/bare-metal-raspberry-pi-minidexed-dx7/
> 
> Kevin.

Thank you so much, Kevin. Exactly such a summary with hints about the hardware and the configuration is what I missed.

**Reply by JoaquinSainz** [reply]
Thank you all, you helped me a lot, I was able to solve the problems.  I found that the best way to store banks in the sysex/voice folders is nnnnnn_BankName.  
Cheers✌️

**Reply by krunchr** [reply]
Since I usually build my circuits on strip boards, a regular schematic would be very helpful for me. 

@diyelectromusic Maybe you can export the schematic from Fritzing and publish it on your website? That would be very nice.

**Reply by diyelectromusic** [reply]
> Maybe you can export the schematic from Fritzing and publish it on your website? That would be very nice.

There is  draft one being discussed here: https://github.com/probonopd/MiniDexed/issues/163.
I also have a schematic of my "DIY HAT" in part 2 on my website.

Kevin

**Reply by krunchr** [reply]
> There is draft one being discussed here: #163. I also have a schematic of my "DIY HAT" in part 2 on my website.
> 
> Kevin

Thank you for the hint.
